frontend/packages/data-portal/app/components/Run/Matrix4x4.tsx

```diff
@@ -1,10 +1,11 @@
+import { TableDataValue } from 'app/types/table'
 import { cns } from 'app/utils/cns'
 
 export function Matrix4x4({
   matrix,
   className,
 }: {
-  matrix: string
+  matrix: TableDataValue
```

Comment:
why this change? it was meant as a string only because the way i encoded the matrix was a flat string delimited by spaces e.g. `1 0 0 0 0 1 0 0 0 0 1 0 0 0 0 1`